Update pricing to reinclude legacy (#17389)
cloudflare/cloudflare-docs: src/content/docs/workers/platform/pricing.mdx

:::

## Legacy Workers Pricing

> If you are a Workers Enterprise customer, you may have opted to remain on the legacy Bundled or Unbound usage models with the following inclusions: 

Reviewer: I think included values is going to be less relevant to ENT customers. Thoughts on just making the table the units we charge for? And keeping that key differentiator is the CPU time limit that bundled has

> If you are a Workers Enterprise customer, you may have opted to remain on the legacy Bundled or Unbound usage models with the following inclusions: 

Reviewer: ```suggestion
If you are a Workers Enterprise customer, you may have opted to remain on the legacy Bundled or Unbound usage models. Bundled and Unbound are billed on the following metrics, rates vary based on your contract: 
```

| Usage Model | Requests  | Duration  | CPU time  |       
|---- | ----- | ------ | ------- |

> | **Bundled** | 10 million included per month | No charge for duration | 50 milliseconds CPU time per invocation  | 

Reviewer: ```suggestion
| **Bundled** | $ / MM requests | No charge for duration | 50 milliseconds CPU time per invocation  | 
```

> | **Unbound** | 1 million included per month | 400,000 GB-s included per month | 30 seconds of CPU time per invocation <br /> 15 minutes of CPU time per [Cron Trigger](/workers/configuration/cron-triggers/) or [Queue Consumer](/queues/configuration/javascript-apis/#consumer) invocation |

Reviewer: ```suggestion
| **Unbound** | $ / MM requests | $ / MM GB-s | 30 seconds of CPU time per invocation <br /> 15 minutes of CPU time per [Cron Trigger](/workers/configuration/cron-triggers/) or [Queue Consumer](/queues/configuration/javascript-apis/#consumer) invocation |
```

> If using legacy pricing, you may switch between the Bundled and Unbound usage models. 

Reviewer: ```suggestion
```

I'd remove since not something we want to encourage.

**To change your usage model at the individual Worker level:**

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
2. In Account Home, select **Workers & Pages**.

> 3. In **Overview**, select your Worker > **Settings** > **Usage Model**.

Reviewer: ```suggestion
3. In **Overview**, select your Worker > **Settings** > **General** > **Pricing**.
```
